packages/app/client/src/ui/organisms/ProjectVerticalCard.js

```diff
@@ -5,109 +5,174 @@ import CardContent from '@material-ui/core/CardContent'
+  root: {
+    [theme.breakpoints.only('md')]: {
+      maxWidth: 285
     },
-    cardContent: {
-      paddingLeft: 0
-    },
-    title: {
-      fontStyle: 'normal',
-      fontWeight: 600,
-      fontSize: '18px',
-      lineHeight: '140%',
-      marginBottom: 20
-    },
-    details: {
-      overflowWrap: 'break-word',
+    [theme.breakpoints.only('sm')]: {
+      maxWidth: 245
     }
-  })
-)
+  },
+  footer: {
+    paddingTop: 0,
+    paddingLeft: 0
+  },
+  footerText: {
+    fontSize: 13,
+    lineHeight: '180%',
+    fontWeight: 'normal',
+    color: theme.palette.text.primary
+  },
+  code: {
+    fontWeight: 500,
+    marginLeft: 5,
+    textDecoration: 'none'
+  },
+  cardActions: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    paddingLeft: 0
+  },
+  cardContent: {
+    paddingLeft: 0
+  },
+  title: {
+    fontStyle: 'normal',
+    fontWeight: 600,
+    fontSize: '18px',
+    lineHeight: '140%',
+    marginBottom: 20
+  },
+  details: {
+    overflowWrap: 'break-word'
+  },
+  projectButtons: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginTop: '5%'
+  }
+}))
 
-const ProjectVerticalCard = ({ disableActions, ...props }) => {
+const ProjectVerticalCard = ({
+  disableActions,
+  isOwner,
+  maxWidth,
+  ...props
+}) => {
   const classes = useStyles()
   const trimmedDetails = useTrimmedText({ text: props.details, max: 134 })
-  
+
   return (
     <CardWithMedia
       className={classes.root}
-      maxWidth={345}
+      maxWidth={maxWidth}
       maxHeight={509}
       mediaProps={{
         media: 'image',
         src: props.src,
         title: props.imageTitle,
         height: 208,
-        alt: props.imageAlt 
+        alt: props.imageAlt
       }}
       actionAreaContent={
         <CardContent className={classes.cardContent}>
-          <Typography gutterBottom variant="h3" component="h6" className={classes.title}>
+          <Typography
+            gutterBottom
+            variant="h3"
+            component="h6"
+            className={classes.title}
+          >
             {props.name}
           </Typography>
           <Grid item container xs={12}>
-            <Typography variant="caption" component="p" className={classes.details}>
+            <Typography
+              variant="caption"
+              component="p"
+              className={classes.details}
+            >
               {trimmedDetails}
             </Typography>
           </Grid>
         </CardContent>
       }
       {...props}
     >
-      <CardActions className={classes.cardActions}>
-        <LikeButton disabled={disableActions} id={props.id} mr={20} numberOfLikes={props.likesCount} onLike={props.onLike} liked={props.userLiked}/>
-        <CommentsButton disabled={disableActions} onClick={props.openComments} numberOfComments={`${props.commentsCount} comments`} />
-      </CardActions>
+      {isOwner ? (
+        <div>657 likes 67 comments 22.02.2022</div>
+      ) : (
```

Comment:
Does this new code change the cards on the Projects' main page?  It shouldn’t change.